pull_request_template.md

```diff
@@ -1,9 +1,14 @@
-Test plan - (Please fill in how you tested your changes)
-
-Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines. Don't forget to follow our [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution) for any code copied from other projects.
-
-Fill in the release notes towards the bottom of the PR description.
-See [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) for details.
+## Description
+<!--- Describe your changes in detail -->
```

Comment:
Will the PR include HTML comment?  If not, let's just keep them uncommented?

Reply:
PR will not include HTML comments. I have uncommented them.